about.html

```diff
@@ -5,6 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>About</title>
+    <style>
+      body {
+        margin: 0;
```

Comment:
@xDAnkit do we need to add margin here.
Can we do it in common styling

Reply:
Hey @std447 , as we're not aligned to create a separate file hence we decided to go with internal CSS>

Reply:
Done